README.md

```diff
@@ -40,7 +40,7 @@ Options:
   --dev                                                    Flag to turn on the dev mode.  false by default.
   --dev.block-interval <Int32>                             The time interval between blocks. It's unit is milliseconds. Works only when dev mode is on.  10000 (ms) by default. (Default: 10000)
   --dev.reorg-interval <Int32>                             The size of reorg interval. Works only when dev mode is on.  0 by default. (Default: 0)
-  --log-minimum-level <String>                             The log minimum level during standalone execution. (Default: debug)
+  --log-minimum-level <String>                             The log minimum level during headless execution. (Default: debug)
```

Comment:
You MUST update the string in [Program.cs#L114](https://github.com/planetarium/NineChronicles.Headless/blob/a86c323caf2c8f2058590e36371f041f40a90737/NineChronicles.Headless.Executable/Program.cs#L114) if you want update this line.

Reply:
I see, I'll put a PR for that update!